### ISSUE TITLE
[Tizen] View units are not device-independent like all the others

### ISSUE THREAD

**Issue by mattleibow** [xamarin/Xamarin.Forms#2693]
### Description

This is not a bug exactly, but rather a difference in how the platforms were implemented. 

On iOS, Android and UWP, all view dimensions and other units are device-independent. For example, on my Android device, the native view has a width of 1080px, but when it appears in Xamarin.Forms, the view has a width of 360. This is the same for iOS and UWP.

Tizen, however, does not do this, resulting in code that executes differently. In my specific case (SkiaSharp), I calculate the density of the screen by using the `scale = canvasWidth / view.Width`. On all the platforms, the `canvasWidth` is the actual, physical pixels, and the `view.Width` is scaled pixels. As a result, I can use `scale` to draw all my images the same size on any screen/device. However, on Tizen, everything is much smaller.

### Expected Behavior

The view's dimensions to be scaled pixels.

### Actual Behavior

The view's dimensions were raw pixels.

**Comment by mattleibow** [comment]
Aaaaand, I just discovered there is an overload of `Forms.Init(…)` that takes a bool to switch this on.

We may want to do something about this as it may confuse users coming from iOS or Android. Unfortunately, any changes will break all existing users. Docs may be the only way.

**Comment by rookiejava** [comment]
You are right. Unlike android and iOS, Tizen native UI framework doesn't uses divice independent pixel unit as a nature.

As mentioned above, to use device-independent pixel units, you must use 'Forms.Init (application, true)' instead of 'Forms.Init (application)'.

Maybe, this may cause user confusion. However, I think 'Forms.Init()' is a part of platform specific area. You know there are already some differnce across platforms. E.g. android's Forms.Init() need 2 arguments (context and bundle), rather iOS no need argument for Forms.Init().

Of course, I can change the default behavior for using device independent pixel unit from false to true, if everyone want. Or, we can providing this settings (whether device independent pixel unit is use or not) via 'Flags'.

**Comment by mattleibow** [comment]
I think it is fine where the option is now... I just spent a long time before I found it and I am thinking that other new guys will too.

Maybe because Tizen is new, or for Tizen.Forms v3.0, we can switch the default to be on.
